### ISSUE TITLE
Récap brief cassé

### ISSUE THREAD

**Issue by psychopatt** [NextINpact/Next#340]
Mon flux RSS (condensé) ne m'ayant pas donné de récap brief vendredi 26 et aujourd'hui lundi 29, je me suis rendu sur sur le site pour regarder ce qu'il en était. Surprise : sur desktop (Firefox, Chromium), sur la la page d'accueil (rafraîchie en vidant le cache) le lien vers #LeBrief pointe vers le brief de [jeudi dernier (25 avril)](https://next.ink/briefs/lebrief-tiktok-suspend-sa-version-lite-ariane-6-debout-enquete-sur-pegasus-leurope-et-la-violence-numerique-aux-femmes/). Sur mobile, l'onglet Brief montre bien qu'il y a eu des brèves publiées vendredi et lundi. Un souci de génération de récap ?

**Comment by psychopatt** [comment]
(#229 et #226 seraient bien pratiques pour contourner, sur desktop je n'ai pas trouvé de moyen de voir les brèves des autres jours)

**Comment by psychopatt** [comment]
Erf, duplicata de #339 , my bad.

**Comment by Glandos** [comment]
Non, ce n'est pas un duplicata de #339. Le lien de flux `condensedbrief` ne contient plus du tout les brèves. Mais je pense qu'il est amené à disparaître, après les annonces faites dans https://next.ink/135983/6-mois-deja/ #LeBrief n'est plus sensé exister au format « article entier ».

**Comment by psychopatt** [comment]
Yep, j'ai lu l'article ce matin. 

Edit : du coup je réouvre juste pour signaler que le flux "condensé" est dorénavant un piège.
Edit 2 : en relisant les deux derniers messages je me rends compte que je ne suis pas clair du tout, je laisse uniquement l'essentiel.
